test/LANDRegistry.js

```diff
@@ -1098,4 +1097,150 @@ contract('LANDRegistry', accounts => {
       await assertRevert(land.assignNewParcel(0, 3, user, sentByOperator))
     })
   })
+
+  describe('ping', () => {
+    it('should set reactivate authorized', async () => {
```

Comment:
I would have a separate `describe` for:

`setReactivateAuthorized`
`ping`
`setGracePeriod`
`setDeemPeriod`

Reply:
I think that we are using 
```javascript
async function () {}
``` 
everywhere. It would be great if you can follow that